code/modules/surgery/organs/organ_external.dm

```diff
@@ -149,6 +149,17 @@
 	if(HAS_TRAIT(owner, TRAIT_BURN_WOUND_IMMUNE))
 		limb_flags |= CANNOT_BURN
 
+/obj/item/organ/external/proc/remove_limb_flags()
```

Comment:
If you're adding this here, I feel like you should make sure it gets put into all the proper spots (similar to how add_limb_flags is used in a few places), but that'd probably be out of scope.

Reply:
I'll do this in a separate PR once this is in!